### ISSUE TITLE
[Bug]: Documentation page is broken

### ISSUE THREAD

**Issue by shelegdmitriy** [NEARBuilders/near-social-js#21]
### Version

1.0.0

### What platform are you seeing the problem on?

Browser (Brave), Browser (Chrome)

### Description

The [nearbuilders.github.io/near-social-js/](https://nearbuilders.github.io/near-social-js/) link from **About** section leads to broken page

### Steps To Reproduce

1. Open https://github.com/NEARBuilders/near-social-js;
2. Click on the [nearbuilders.github.io/near-social-js/](https://nearbuilders.github.io/near-social-js/) link from **About** section or do the same from https://github.com/NEARBuilders/near-social-js?tab=readme-ov-file#-2-documentation section
3. See the 404 page

### Expected Results

A generated Docusaurus page with all the content as it shown locally using `yarn docs:start` script

### Actual Results

The 404 page
<img width="910" alt="Знімок екрана 2024-06-10 о 19 31 02" src="https://github.com/NEARBuilders/near-social-js/assets/34593263/4b244e11-69be-4347-8046-c1938c9cccf8">

**Comment by kieranroneill** [comment]
@shelegdmitriy I think it hasn't deployed yet. It _should_ deploy once `beta` is merged into `main`.

I say "should" because I have not actually perforned a dry run for the deployment yet 😅

I will keep this issue open until the deployment is successful and working.

Once the [contract-level](https://github.com/NEARBuilders/near-social-js/issues/3#issue-2277400530) functions are done, we can do a `main` release.

**Comment by elliotBraem** [comment]
Resolved
